lib/vue/src/components/input/CInputCheckbox.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div
+      class="d-flex"
+      style="gap: 1.5rem;"
+    >
+      <h6
+        class="mb-0 text-primary"
+        :class="{ 'text-muted': value }"
+      >
+        No
+      </h6>
+      <b-form-checkbox
+        v-bind="$attrs"
+        v-model="value"
```

Comment:
you shouldn't use v-model here since it updates the prop, this is a display only component that means you only emit events when you update and don't v-model